packages/sku/config/webpack/webpack.config.js

```diff
@@ -99,7 +100,7 @@ const makeWebpackConfig = ({
     // Libraries should always have the same file name
     // for the main chunk unless we're building for storybook
     if (isLibrary && isMainChunk) {
-      return libraryName;
+      return libraryFile ?? libraryName;
```

Comment:
Could probably remove/reword the comment here as it's not longer true.

Reply:
In this case having the same file name referrers to using a name from the config rather than allowing WebPack to generate it, potentially with a content hash, so this comment is still relevant.